src/main/java/com/mock/preparation/ds/FabonaciiSeriesInArray.java

```diff
@@ -0,0 +1,21 @@
+package mock.preparation.ds;
+
+import java.util.Scanner;
+
+public class FabonaciiSeriesInArray {
+    public static void main(String[] args) {
+        int n1 = 0, n2 = 1, n3, num = 0;
+        System.out.println("Enter the given number: ");
+        Scanner scanner = new Scanner(System.in);
+        num = scanner.nextInt();
+        for (int i = 0; i <= num; i++) {
+            if (num >= 0) {
```

Comment:
does output of this program print seriies as 0, 1 , 1,  2,3 5, 8, 13 ....? I dont think so